src/styles/StyleUtils.ts

```diff
@@ -192,17 +186,10 @@ function getAvatarSize(size: AvatarSizeName): number {
 /**
  * Return the height of magic code input container
  */
-function getHeightOfMagicCodeInput(): ViewStyle {
+function getHeightOfMagicCodeInput(styles: typeof defaultStyles): ViewStyle {
```

Comment:
```suggestion
function getHeightOfMagicCodeInput(styles: ThemeStyle): ViewStyle {
```

We can use `ThemeStyle` for this, wdyt about applying to the other functions?

Reply:
I'll try, for one of the previous PR I tried it somewhere else, but TS was not happy with that

Reply:
Yes, let's use `ThemeStyle` type from `@styles/styles`. Also, can we please rename it to `ThemeStyles` to match other types of this kind

Reply:
@chrispader you mean to rename the type?